static/js/components/ranking_unit.tsx

```diff
@@ -137,14 +139,18 @@ export function RankingUnit(props: RankingUnitPropType): JSX.Element {
     props.isHighest,
     props.numDataPoints
   );
-
   return (
     <div
       className={"ranking-list " + ASYNC_ELEMENT_CLASS}
       ref={props.forwardRef}
     >
       <div className="ranking-header-section">
-        <h4>{props.title}</h4>
+        <h4>
```

Comment:
nit: consider extracting this loading header into a shared component to be used by chart_tile.tsx, ranking_tile.tsx, and here to make sure it is always consistent if changes are made in the future?

Reply:
done